### ISSUE TITLE
Option to disable deletion of sabnzbd history

### ISSUE THREAD

**Issue by kooimens** [Hellowlol/HTPC-Manager#85]
I thought it shouldn't be too hard to do. Tried it, but wasn't succesful. I would like to have an option within the sabnzbd module to hide/disable the ability to remove a sab history item. 
![1](https://cloud.githubusercontent.com/assets/3150962/6734670/cb72ff7c-ce59-11e4-8134-19f1ee9e6489.jpg)

**Comment by Hellowlol** [comment]
I dont know if im gonna add this, if im gonna do that it would be so many options... One possible option is tp add a more restrict user.

**Comment by kooimens** [comment]
I agree in same way. However, the more options, the better. In the end it should be regulated on the user permissions level. As I'm having all my apps in contact with my windows active directory, I don't make use of the user functionality of htpc. The active directory takes care of all the authentication and authorization of my users. And of course, build-in ldap user functionality in htpc would be overkill:D. Some possible solutions:

1. Run in kiosk mode. The maraschino project makes use of this. By enabling this option on the command line, the app can only be 'viewed'. You can think of a kiosk mode or some kind of a non-deletion mode, where stuff can only be added, not deleted. With this you can run multiple instances (on different ports of course) of the same htpc installation and settings, where the one without the kiosk/not deletion mode can only be viewed by the admin.

2. Get the username from the http header and determine user rights. Don't know if this possible and we should of course warn the user for any security threads. With this you can still use an external login app. I can just add the users in htpc and give them a random password. When connecting to htpc, htpc will look for a username in the http header. If it matches a user then allow access and give him the appropriate user role.

**Comment by Hellowlol** [comment]
1. I have already made a kiosk mode, but i did it for https://github.com/madclicker/PyTunes.
2. This is loads of work with imo small benefits. I will not do anything like this.
3. Keep testing the https://github.com/Hellowlol/HTPC-Manager/tree/addnewuser so it get get added to master

**Comment by henkjanvries** [comment]
ive done some custom restrictions in the modules. using the restricted_user mod this works for me for 95% of the time.
is there a way to set these in the setting pages of the modules? i mean, would it be hard to add these option to the settings pages of the modules.

**Comment by Hellowlol** [comment]
There isnt a way, and yes it would be major pita. I will not add on that

**Comment by henkjanvries** [comment]
fair point! keep up the good work
